### PR TITLE
fixed bug with symbol money(K/M)

### PR DESCRIPTION
**Pull request by Pawel-Szydlo**

### CODE REVIEW

**Review by wojciech-cichocki — CHANGES_REQUESTED** [review]
Maybe it will be a good idea to extract K/M symbol as util function.

**Review by awojciak — COMMENTED** [review]
kod ok, ale zgodzę się z tym, co zaproponował Wojtek

File: src/components/Cards/ValueCard.tsx [hunk 1]
```diff
@@ -19,47 +16,28 @@ export interface IProps {
 }
 export const ValueCard: React.FC<IProps> = ({ name, value, sign, hint, onClick }) => {
   const classes = useStyles()
+  console.log(Number(value))
```
Comment (wojciech-cichocki) [inline src/components/Cards/ValueCard.tsx:19]: Looks fine. Remove this console.log().

You should also configure your identity in git: `git config user.email "MY_NAME@example.com"`

**Review by wojciech-cichocki — APPROVED** [review]

File: src/components/Cards/ValueCard.tsx (hunk 1 above)
Comment (Pawel-Szydlo) [reply src/components/Cards/ValueCard.tsx]: ok, i deleted console.log and identified the email. Merge with dev?

Comment (wojciech-cichocki) [reply src/components/Cards/ValueCard.tsx]: sure

**Pull request merged**